packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -275,15 +277,21 @@ export abstract class Axis<S extends Scale<D, number, TickInterval<S>> = Scale<a
         this.assignCrossLineArrayConstructor(this._crossLines);
 
         let previousSize: { width: number; height: number } | undefined = undefined;
-        this.destroyFns = [
+        this.destroyFns.push(
```

Comment:
There's a hover listener on line 260 that was not being destroyed as it was being dumped by this. Probably wasn't causing any perf issues though.